faq/faqs-business-logic.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "FAQs: {{site.bs-bl}}"
+description: "An overview of the most common {{site.tp}} {{site.bs-bl}} operations presented in a Q&A format."
+type: faq
+page_title: "FAQs: Common operations in {{site.bs-bl}}"
+slug: faq-business-logic-data-operations
+position:
+tags: backendservices, cloud-code
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Code for Data, Cloud Functions</td>
+  </tr>
+</table>
+
+## Introduction
+
+This article answers questions about common operations with the {{site.bs-bl}} service of {{site.tp}}. They mainly revolve around using the `request` and `response` objects to make application-critical checks and also performing common operations.
+
+If not stated otherwise, the content is applicable both to {{site.bl-data}} and {{site.bl-func}}.
+
+* [Can I determine which user account is making the request?](#can-i-determine-which-user-account-is-making-the-request)
+* [How do I get the body of the incoming request?](#how-do-i-get-the-body-of-the-incoming-request)
+* [How do I get the result status of the request?](#how-do-i-get-the-result-status-of-the-request)
+* [Can I apply filtering criteria before returning data?](#how-to-apply-filtering-criteria-before-returning-data)
+* [How do I determine the Id of the requested item?](#how-do-i-determine-the-id-of-the-requested-item)
```

Comment:
We want to refer to the Id explicitly as your data items will have an "Id" and not "ID". Technically, the notion in BS is called 'Id'. That is why it was preferred over the generic "ID".